lib/js/jsffi.nim

```diff
@@ -94,7 +102,7 @@ proc hasOwnProperty*(x: JsObject, prop: cstring): bool
   {. importcpp: "#.hasOwnProperty(#)" .}
   ## Checks, whether `x` has a property of name `prop`.
 
-proc jsTypeOf*(x: JsObject): cstring {. importcpp: "typeof(#)" .}
+proc jstypeof*(x: JsObject): cstring {. importcpp: "typeof(#)" .}
```

Comment:
camelCase is valid for this case, not sure why you've changed it.

Reply:
I made everything consistent. The other definitions were following this scheme, but I agree that the whole module could have followed NEP-1 from the start.